### PR TITLE
Added default folder and more troubleshooting docs

### PR DESCRIPTION
**Pull request by zneix**
We refer to "appdata folder" as "Chatterino folder" and there are also troubleshooting steps for Chatterino not saving user settings.
Added missing dots to few sentence endings as well.

### CODE REVIEW

File: docs/Help.md [hunk 1]
```diff
@@ -68,18 +68,28 @@ If you're getting the `Login expired for user <user>! Try adding your account ag
...
-For **Linux**:
-Delete the `$HOME/.local/share/chatterino/Cache` folder: `rm -rf "$HOME/.local/share/chatterino/Cache"`
+On **Mac**:
+`$HOME/Library/Application Support/chatterino`
 
-For **Mac**:
-Delete the `$HOME/Library/Application Support/chatterino/Cache` folder: `rm -rf "$HOME/Library/Application Support/chatterino/Cache"`
+### How do I delete the Chatterino cache / settings?
+Navigate to [Chatterino folder](#where-is-chatterino-folder-located) and remove corresponding folders.
+
```
Comment (alazymeme) [inline docs/Help.md:88]: No need for this extra space

File: docs/Help.md [hunk 2]
```diff
@@ -68,18 +68,28 @@ If you're getting the `Login expired for user <user>! Try adding your account ag
...
-2. Turn on `Always on top` in Chatterino
-3. Click the 3 dots up the top of a channel split and click `Popup`
-4. Move and resize the chat to overlay on top of the standard Twitch chat
+1. Add a dock into OBS for standard Twitch chat.
+2. Turn on `Always on top` in Chatterino.
+3. Click the 3 dots up the top of a channel split and click `Popup`.
+4. Move and resize the chat to overlay on top of the standard Twitch chat.
 
-### How do I delete the Chatterino cache?
+### Where is Chatterino folder located?
```
Comment (alazymeme) [inline docs/Help.md:76]: There are technically two chatterino folders. Maybe `Where is my Chatterino data folder located?` or something similar.

File: docs/Help.md [hunk 3]
```diff
@@ -68,18 +68,28 @@ If you're getting the `Login expired for user <user>! Try adding your account ag
...
 
-For **Linux**:
-Delete the `$HOME/.local/share/chatterino/Cache` folder: `rm -rf "$HOME/.local/share/chatterino/Cache"`
+On **Mac**:
+`$HOME/Library/Application Support/chatterino`
 
-For **Mac**:
-Delete the `$HOME/Library/Application Support/chatterino/Cache` folder: `rm -rf "$HOME/Library/Application Support/chatterino/Cache"`
+### How do I delete the Chatterino cache / settings?
+Navigate to [Chatterino folder](#where-is-chatterino-folder-located) and remove corresponding folders.
```
Comment (alazymeme) [inline docs/Help.md:87]: I'm not sure about this, it's making a simple step more complicated for no reason. Where a simple troubleshooting step such as "Delete your Chatterino cache" can be used and easily found, they now have to go to two different locations to find out what they have to delete.

**Review by alazymeme — CHANGES_REQUESTED** [review]

File: docs/Help.md (hunk 2 above)
Comment (pajlada) [inline docs/Help.md:76]: phrasing, change to "the Chatterino folder" or "my Chatterino folder" - not sure which is best, but as long as we're consistent it should be all good

File: docs/Help.md (hunk 3 above)
Comment (pajlada) [inline docs/Help.md:87]: make this babyproof

**Review by pajlada — CHANGES_REQUESTED** [review]

**Review by pajlada — APPROVED** [review]

**Pull request merged**